### ISSUE TITLE
Distrobox's desktop file don't open with terminator, only konsole

### ISSUE THREAD

**Issue by brunofavs** [brunofavs/my_arch_issues#2]
```
[Desktop Entry]
Name=Ubuntu
GenericName=Terminal entering Ubuntu
Comment=Terminal entering Ubuntu
Categories=Distrobox;System;Utility
Exec=/usr/bin/distrobox enter  ubuntu
Icon=/home/bruno/.local/share/icons/distrobox/ubuntu.png
Keywords=distrobox;
NoDisplay=false
Terminal=true
TryExec=/usr/bin/distrobox
Type=Application
```
![2024-03-2416-39-21-ezgif com-video-to-gif-converter](https://github.com/brunofavs/my_arch_issues/assets/31831043/26aa7010-a36b-4314-994f-facf121cab49)

But with konsole set as default opens fine from the desktop

![2024-03-2416-59-25-ezgif com-video-to-gif-converter](https://github.com/brunofavs/my_arch_issues/assets/31831043/e8d2176c-a13a-4d46-b5da-be2ad0966232)

Also manually through terminator also opens fine.
![image](https://github.com/brunofavs/my_arch_issues/assets/31831043/77a4bd74-3b40-407f-b3be-f42cf52507f4)

**Comment by brunofavs** [comment]
Maybe I should mess arround with Terminator's profiles and custom shortcuts

**Comment by brunofavs** [comment]
I got a workaround working almost perfectly.  The app still doesn't open but now a shortcut does, which is much better.

Basically I have a new profile set on terminator that when opened executes a custom zshrc (on top of the main zshrc) which will contain shell configurations specific to ubuntu.

![image](https://github.com/brunofavs/my_arch_issues/assets/31831043/8dd3eba8-85c9-4af4-a62e-239328b749a2)


Also in this zshrc the ubuntu container is initialized : 
![image](https://github.com/brunofavs/my_arch_issues/assets/31831043/96dd99ea-cb30-456d-b84d-c74978af99a2)

This will after also include this I only want for the ubuntu such as the ROS environment variables and sources.

To execute it separately, I configured a custom keyboard shortcut : 

And it works really well :)))) 

![image](https://github.com/brunofavs/my_arch_issues/assets/31831043/8800eaa3-934b-45b4-a86f-91555959fc66)

It also has different colors so that I visually aid me to know if I'm on the correct terminal
